assets/css/src/components-form-editor/_template-selection.scss

```diff
@@ -0,0 +1,3 @@
+.template-selection {
+  margin: 30px;
```

Comment:
It's probably better to use `$grid-gap` or `$grid-gap-large` but I guess styling will come with another ticket, right?